### DISCUSSION TITLE
Android app crashing

### DISCUSSION THREAD

**Discussion by timothybest** [advplyr/audiobookshelf#3281] [Q&A]
Started today with latest version of the android app. The app crashes on every file I try to play. I tried deleting the app and reinstalling but it didn't fix it.

**Comment by nichwall** [comment]
What app and server version? Are you on 0.9.76?

**Comment by timothybest** [comment]
app 0.976-beta / server 2.12.3
It was just fixed in an updated release of the app that came out yesterday.

**Comment by TheOtherOtherJohn** [comment]
Updating th app fixed this for me, however my container no longer reports its version. Still works but waiting for something bad to happen. :D
